com.gluonhq.scenebuilder.yml

```
@@ -0,0 +1,63 @@
+app-id: com.gluonhq.scenebuilder
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: SceneBuilder
+finish-args:
+  - --socket=x11
+  - --share=network
+cleanup:
+  - /lib/debug/
+  - /share/info/*
+  - /share/locale/*
+  - /share/man/*
+  - /share/runtime/locale/*
+  - 
+modules:
+  - name: cpio
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/cpio/cpio-2.12.tar.gz
+        sha256: 08a35e92deb3c85d269a0059a27d4140a9667a6369459299d08c17f713a92e73
+      - type: patch
+        path: cpio-2.13-mutiple-definition.patch
+
+  - name: rpm2cpio
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/ruda/rpm2cpio/archive/2017.tar.gz
+        sha256: d266a4f49e65b5d05956502549ceb1d5671f927375ff71299069e6d92363297a
+      - type: patch
+        path: rpm2cpio.patch
+    build-commands:
+      - install -D rpm2cpio.py /app/bin/rpm2cpio
+
+  - name: SceneBuilder
+    buildsystem: simple
+    only-arches:
+    - x86_64
+    build-commands:
+    - rpm2cpio SceneBuilder-18.0.0.rpm | cpio -idmv
```

Just below you're setting `filename` to `SceneBuilder.rpm`. This is not going to build without changing it here as well.

---

and suggest:
```suggestion
    - rpm2cpio SceneBuilder*.rpm | cpio -idmv
```

Non-withstanding my other suggestion to use the `.deb`